### DISCUSSION TITLE
Problema de responsividade

### DISCUSSION THREAD

**Discussion by 404jv** [404jv/dev-landia#88] [Q&A] [answered]
É um problema simples, mas é complicado de resolver e quanto mais demorar mais complicado fica. Então, acho uma boa os mago do React (@FixRuan @jvolima) aí tentar dá um jeito nisso.

Por exemplo, o @jvolima refatorou a aba de medalhas e agora está assim pra mim:
<img width="180" src="https://user-images.githubusercontent.com/53544964/178289065-d6b8b084-231a-4f48-8889-d58794ea4c40.png"/>
Porém, no celular dele está suave e aposto que no emulador fica dboa também.

A mesma coisa está acontecendo com a Home do app. Eu fiz de um jeito que ficou bom no meu e do Jão quebrou, quando ele arrumou, o meu que quebrou. Tá uma loucura.

**A pergunta é**: Alguma ideia de onde começar adicionando responsividade e quais libs vamos usar pra resolver o B.O (se é que vai usar alguma)

**Comment by jvolima** [comment, marked as answer]
eu vou corrigir esses problemas agora nas férias, vou testar algumas coisas mas acredito que dê para arrumar sim
